### PR TITLE
Merge python adaptor sites with main PORI site

### PR DESCRIPTION
**Pull request by creisle**
- Add IPR & GraphKB subsections to main PORI docs
- Add documentation page for how to load data into GraphKB
- Add FAQs page
- add script to include python adaptor documentation into the main site

Main page now should look like this

image

### CODE REVIEW

**Comment by elewis2** [comment]
Under 'Developer reference', IPR and GraphKB have incorrect capitalizion

**Comment by creisle** [comment]
> Under 'Developer reference', IPR and GraphKB have incorrect capitalization

agreed, they should be lower-case to match the python name but there isn't a good way around that currently unfortunately. mkdocs automatically capitalizes them and the directories itself are auto-generated so I can't add a .pages file

File: docs/graphkb/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# About
+
+GraphKB is a graph-based implementation of a cancer knowledge base. Unlike other knowledge base projects,
```
Comment (elewis2) [inline docs/graphkb/index.md:3]: this sentence is a bit weird. maybe 'graphkb is unique among knowledge base projects in its inclusion of ontology relations....'?

File: docs/install.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 # Install with Docker
 
-The simplest way to get the entire platform up an running from scratch is using docker. Most of the servers are auto-started together with docker-compose but the keycloak container must be started and configured on its own first. The instructions below set up the platform with HTTP and then use a reverse proxy to pick up the ports. This way you can omit the proxy step and run the platform with http when initialling setting up and testing.
+Since PORI is a production-ready, instituion-level, scaleable platform, the simplest way to get the entire platform up an running from scratch is using docker. Most of the servers are auto-started together with docker-compose but the keycloak container must be started and configured on its own first. The instructions below set up the platform with HTTPS and then use a reverse proxy to pick up the ports. This way you can omit the proxy step and run the platform with http when initialling setting up and testing.
```
Comment (elewis2) [inline docs/install.md:4]: instituion -> institution; up an running -> up and running

Comment (creisle) [reply docs/install.md]: fixed

File: docs/graphkb/index.md (hunk 1 above)
Comment (creisle) [reply docs/graphkb/index.md]: yes that's better, changed

**Review by elewis2 — APPROVED** [review]

**Pull request merged**